### ISSUE TITLE
Accuracy is 0 when trained on two clients

### ISSUE THREAD

**Issue by mustansarsaeed** [OpenMined/KotlinSyft#292]
## Question
Hi . I am in a strange issue. I have simple model:

```
class SyntheticNet(nn.Module):
    def __init__(self):
        super(SyntheticNet, self).__init__()
        self.fc1 = nn.Linear(60, 5)

    def forward(self, x):
        x = self.fc1(x)
        x = nn.functional.relu(x)
        return x
``` 

 When I train on single client it reaches to accuracy 90% in first 2-3 rounds. When I train the same model on 2 clients that accuracy is near to 0.  

Can it be the case that we are setting model parameters in ```training_plan``` and one client's model parameters are overriden by other client's parameter that's why accuracy is not improving?

**Comment by vkkhare** [comment]
Is this sorted @mustansarsaeed ? I think we discussed this on slack

**Comment by mustansarsaeed** [comment]
Thanks @vkkhare . Yes, it got fixed. There were some other issues as well during the averaging.
